Docs/Definitions.md

```diff
@@ -1,55 +1,61 @@
-## Sidecar pattern
-
-Extends the functionality of the main container with a strong coupling to a sidecar container. A sidecar is a utility container in a pod with the purpose to support thte main container. Standalone sidecars don't serve any purpose, must be paired with one or more containers. Generally are reusable and can be paired with numerous ypes of main containers
+# Definitions
 
 ## Ambassador pattern
 
-The ambassador pattern is a useful way to connect containers with the outside world. An ambassador container is essentially a proxy that allows other containers to connect to a port on localhost while the ambassador container can proxy these connections to different environments depending on the cluster's needs.
+The ambassador pattern is a useful way to connect containers with the outside world. An ambassador container is essentially a proxy that allows other containers to connect to a port on localhost while the ambassador container can proxy these connections to different environments depending on the cluster's needs. The ambassador pattern is an example of the **Sidecar pattern**.
 
```

Comment:
Having looked back into this I think the ambassador and sidecar pattern are separate and it would technically be incorrect to state 'The ambassador pattern is an example of the Sidecar pattern'. Reading the definitions again, a sidecar pattern is an additionally attached pod that extends the functionality of the pod. However the ambassador pattern is purely to act as a proxy to simplify the access of external servers but has no impact on the actual functionality of the pod itself

Reply:
As discussed IRL, this is a bit of a grey area but the Ambassador pattern is an example of the Sidecar pattern (see e.g. https://docs.microsoft.com/en-us/azure/architecture/patterns/sidecar )